website/routes.py

```diff
@@ -882,6 +882,24 @@ def make_url_map(app):
             project_views.register.node_registration_embargo_disapprove,
             OsfWebRenderer('error.mako', trust=False)
         ),
+        Rule(
```

Comment:
We had discussed putting tokens in the querystring when the user claiming code was being developed and we decided against it, though I do not remember the exact reason. @JeffSpies Do you remember?

In any case, the current routes that use tokens put the token in the URL path.